platform/npm/tests/snfe.js

```diff
@@ -128,4 +128,53 @@ describe('SNFE', () => {
             ]));
         });
     });
+
+    describe('Serialization', () => {
+        beforeEach(async () => {
+            const globals = { URL, setTimeout, clearTimeout };
+
+            const { StaticNetFilteringEngine } = await createWorld('./index.js', { globals });
+
+            engine = await StaticNetFilteringEngine.create();
+        });
+
+        it('should not reject with no lists', async () => {
+            await engine.useLists([]);
+
+            await engine.serialize();
+        });
+
+        it('should not reject with one empty list', async () => {
+            await engine.useLists([
+                { name: 'easylist', raw: '' },
+            ]);
+
+            await engine.serialize();
```

Comment:
@gorhill ideally we would check that the serialized output is correct but I'm not sure you want to tie yourself into this now. The only condition for now is that the result of `serialize()` can be consumed by `deserialize()` and nothing else.

Reply:
>The only condition for now is that the result of `serialize()` can be consumed by `deserialize()` and nothing else.

We'll check this later in the tests for deserialization.

You can always write more of your own tests.